### DISCUSSION TITLE
Error in r$status_code : $ operator is invalid for atomic vectors

### DISCUSSION THREAD

**Discussion by EvaChung6** [MRCIEU/TwoSampleMR#461] [Q&A]
I am trying to extract the data using TwoSampleMR package, but it showed ERROR. The function worked well before.

> exposure_dat <- extract_instruments(exposure_id, p1=5e-08, clump=TRUE)
Error in r$status_code : $ operator is invalid for atomic vectors

Is there any way to solve this? Thanks.

**Comment by yqhu96** [comment]
One possible cause is a loss of connection to your server or to the other server

**Comment by alicechen14** [comment]
this happened to me as well. Still didn't know how to fix this
